docs/production/authentication/microsoft.mdx

```diff
@@ -181,6 +181,24 @@ You may need to restart your Mage server for the changes to take effect.
 
 </AccordionGroup>
 
+### Configurable environment variables
+To configure the environment for Microsoft Azure Cloud and Microsoft Azure Gov Cloud sign-in, 
+you need to set the following environment variables:
+1. You can specify the `AZURE_HOST` URLs based on your environment:
+```
+AzurePublicCloudHostUrl = "https://login.microsoftonline.com/";
```

Comment:
@colefreeman Should the semicolon be there at the end of this line and the next few lines?

Reply:
Addressed and tested in mintlify dev env.
<img width="873" alt="Screenshot 2024-08-22 at 3 29 50 PM" src="https://github.com/user-attachments/assets/4c9548dd-796b-4e22-aed6-b7980e7fa99f">